docusaurus/docs/iOS/controllers/controllers-overview.md

```diff
@@ -2,75 +2,27 @@
+The `StreamChat` framework ships with controllers and delegates that you can use to build your own views.
 
----
-#### `CurrentChatUserController`
-Used for operations related to the current user, like setting the current user, observing its changes, etc.
-
-*Typical usage:* Current user profile screen, a log-in screen, ...
-
-&nbsp;
-
-#### `ChatConnectionController`
-Used for observing a connection status of the system and handling the connection manually if `config.shouldConnectAutomatically` is set to false.
-
-*Typical usage:* connection status banner (connecting/connected), ...
-
-&nbsp;
-
-#### `ChatUserController`
-Used for operations related to any user of a chat, like muting, getting user info, and observing its changes.
-
-*Typical usage:* User profile screen, quick actions on a user ("Mute user"), ...
-
-&nbsp;
-
-#### `ChatUserListController`
-Used to get a list of all users in the chat app. You can also specify `UserListQuery` to filter the users. Supports pagination.
-
-*Typical usage:* Show list of all users in the app.
-
-&nbsp;
-
-#### `ChatChannelListController`
-Used to get a list of all channels in the app matching the provided `ChannelListQuery`. Supports pagination.
-
-*Typical usage:* Show a list of channels (the main page of most chat apps)
-
-&nbsp;
-
-#### `ChatChannelController`
-Used for operations related to a channel, like getting all messages, sending new messages, but also adding/removing members or editing the channel details and deleting it.
-
-*Typical usage:* Messages screen, channel quick actions ("Delete channel", etc.
-
-&nbsp;
-
-#### `ChatChannelMemberController`
-Used for operations related to a member of a channel, like banning, getting member info, and observing its changes.
-
-*Typical usage:* Member profile screen, quick actions on a member ("Ban user"), ...
-
-&nbsp;
-
-#### `ChatChannelMemberListController`
-Used to get a list of all members of a channel. You can also specify `MemberListQuery` to filter the members. Supports pagination.
-
-*Typical usage:* Show list of all members in the channel.
-
-&nbsp;
-
-#### `ChatMessageController`
-Used for operations related to a message, like deleting the message, adding reacting, and observing its changes.
+Each controller exposes API functionality and supports delegation, controllers their delegates are documented here based on the kind of data they control and allow you to observe. You can find examples on how to build your own view as well.
 
-*Typical usage:* Message detail screen, quick actions on a message ("Add Reaction"), ...
+## Using Controllers
 
-&nbsp;
+Controllers expose functionality via event subscriptions and API calls. In most cases you need to call the `synchronize` method on your controller instance. This is done to 
+avoid side-effects from initialization and to make sure that the controller is ready to be used.
 
```

Comment:
- [ ] Add a code block generic example of calling synchronize with a controller.

Reply:
Writing down things as you go is dangerous and sometime you tell somebody to do something they already did. Everybody learn from my lesson. 🤦